html/rendering/non-replaced-elements/form-controls/button-style.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<title>default style on buttons</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div class="tests">
+ <input type="reset">
+ <input type="button">
+ <input type="submit">
+ <input type="color">
+ <button></button>
+</div>
+<script>
+[].slice.call(document.querySelectorAll('.tests > *')).forEach(el => {
```

Comment:
nit: querySelectoAll should be iterable, so you can call forEach on it.

Reply:
Yeah but it hasn't been for long (not sure it works cross-browser yet?), I didn't want to rely on newish features here.

Reply:
Fair :)